addons/stock/models/res_users.py

```diff
@@ -13,7 +13,7 @@ def _get_default_warehouse_id(self):
         if self.property_warehouse_id:
             return self.property_warehouse_id
         # !!! Any change to the following search domain should probably
-        # be also applied in sale_stock/models/sale_order.py/_init_column.
+        # be also applied in stock/models/stock_warehouse.py/_init_column.
```

Comment:
This comment shouldn't be modified, since you didn't move the `_init_column` override ;)